include/dgl/runtime/packed_func.h

```diff
@@ -398,6 +398,10 @@ class DGLPODValue_ {
     DGL_CHECK_TYPE_CODE(type_code_, kDLInt);
     return value_.v_int64 != 0;
   }
+  operator std::string() const {
+    DGL_CHECK_TYPE_CODE(type_code_, kStr);
+    return std::string(value_.v_str);
+  }
```

Comment:
I double checked `DGLArgValue` and `DGLRetValue`. They all have corresponding string support (https://github.com/dmlc/dgl/blob/master/include/dgl/runtime/packed_func.h#L489 and https://github.com/dmlc/dgl/blob/master/include/dgl/runtime/packed_func.h#L578). Why we need this in the base class `DGLPODValue_`? `DGLPODValue_` shouldn't be used directly.

Reply:
You are right. I misunderstood the way of using this part.